codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -978,7 +978,7 @@ protected HeaderDateFormat initialValue() {
                     }
                 };
 
-        static HeaderDateFormat get() {
+        public static HeaderDateFormat get() {
```

Comment:
just a reminder to my self ... instead of exposing this I may want to investigate using the date parsing from https://github.com/netty/netty/pull/6018.